dateutil/test/test_rrule.py

```diff
@@ -2859,6 +2889,56 @@ def testStrValueDate(self):
         self.assertEqual(list(rr), [datetime(1997, 9, 2, 0, 0, 0),
                                     datetime(1998, 9, 2, 0, 0, 0)])
 
+    def testStrSkipBackward(self):
+        rr = rrulestr("DTSTART:20141231T000000\n"
```

Comment:
I'm assuming this is a codecov bug, but for some reason this test is not being reported as reached by codecov...

Reply:
It's my mistake. I used the same name for two tests...